<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedual</string>
  </property>
  <widget class="QGroupBox" name="incident_groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>250</width>
     <height>580</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>当前事项</string>
   </property>
   <widget class="QPushButton" name="incident_add_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="incident_verticalLayout">
     <item>
      <widget class="QListWidget" name="incident_listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="incident_button_horizontalLayout">
       <item>
        <widget class="QPushButton" name="incident_edit_pushButton">
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="incident_cancel_pushButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="single_groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>250</width>
     <height>580</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>编辑</string>
   </property>
   <widget class="QLabel" name="note_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择/新建一个项目</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="confirm_pushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_pushButton">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QWidget" name="incident_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="incident_gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="incident_name_lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="incident_date_label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>截至时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="incident_important_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>重要事件</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="incident_date_Edit">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="incident_name_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>项目名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="incident_important_radioButton">
        <property name="text">
         <string>重要</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="task_widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>196</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="task_gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="task_date_label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>截至时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="task_load_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>总任务量</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateEdit" name="task_date_Edit">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="task_important_label">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>重要程度</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="task_load_spinBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="task_name_lineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="task_important_spinBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="task_name_label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>项目名称</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="task_groupBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>250</width>
     <height>580</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>当前任务</string>
   </property>
   <widget class="QPushButton" name="task_add_pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="task_verticalLayout">
     <item>
      <widget class="QListWidget" name="task_listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="task_horizontalLayout">
       <item>
        <widget class="QPushButton" name="task_edit_pushButton">
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="task_cancel_pushButton">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
